src/main/java/uk/gov/ons/census/casesvc/utility/RedactHelper.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.ons.census.casesvc.utility;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+
+public class RedactHelper {
+  private static final String REDACTION_FAILURE = "Failed to redact sensitive data";
+  private static final String REDACTION_TARGET = "setUac";
+  private static final String REDACT_TO_THIS_VALUE = "REDACTED";
+  private static final ObjectMapper objectMapper = ObjectMapperFactory.objectMapper();
+
+  public static Object redact(Object rootObjectToRedact) {
```

Comment:
Could be named explicitly `redactUacs`

Reply:
deliberately not in case we want this to redact more than just UACs